django/pyrog/api/serializers/import_export.py

```diff
@@ -196,6 +196,16 @@ class Meta:
             "filters",
         ]
 
+    def validate(self, data):
+        request = self.context["request"]
+        auth_token = request.session.get("oidc_access_token")
+        fhir_api = HapiFhirAPI(auth_token)
+        try:
+            data["definition"] = fhir_api.get(f"/StructureDefinition/{data['definition_id']}")
+        except Exception as e:
+            raise serializers.ValidationError({"definition": str(e)})
```

Comment:
```suggestion
            raise serializers.ValidationError({"failed to fetch definition": str(e)})
```

Reply:
nop the format should be "attribute_name": "some error message"